Realm/Realm/Native/SynchronizationContextScheduler.cs

```diff
@@ -0,0 +1,150 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
```

Comment:
Ah... I miss the good old days too 🤤

```suggestion
// Copyright 2020 Realm Inc.
```